llvm/utils/merge-json.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
Do we need LLVM copyright here?

Reply:
Don't think so, none of the other utility scripts I saw had one.